content/en/docs/about/overview.md

```diff
@@ -9,43 +9,71 @@ weight: 1
+You don’t have to manage signing keys, and sigstore services never obtain your private key. The public key that a sigstore client creates gets bound to the issued certificate, and the private key is discarded after a single signing.
+
+After the client signs the artifact, the artifact's digest, signature and certificate are persisted in Rekor, an immutable, append-only transparency ledger, so that signing events can be publicly audited. Identity owners can monitor the log to verify that their identity is being properly used. This also timestamps the signing event, so that the short-lived certificate can be later verified. 
+
+For verifying an artifact, a sigstore client will verify the signature on the artifact using the public key from the certificate, verify the identity in the certificate matches an expected identity, verify the certificate's signature using Sigstore's root of trust, and verify proof of inclusion in Rekor.
+
+For more information on the modules that make up Sigstore, see [Toolling](/docs/about/tooling/)
+
+## How to use Sigstore
 
-## About the project
+To use sigstore, you must first install the product. See the [Installation](docs/system_config/installation/)  instructions. You can then pick the subject matter you wish to learn about from the menu items on the left. For a quick introduction, you can try using one of the links below:
 
-Sigstore is a Linux Foundation project backed by Google, Red Hat and Purdue University. We provide a public good, non-profit service to improve the open source software supply chain by easing the adoption of cryptographic software signing.
+* To get a quick view of how to use the program see [Quick Start](/docs/signing/quickstart/)
+* To learn how to work with blobs, see [sign a blob](docs/signing/signing_with_blobs/)
+* To learn how to work with containers, see [sign a container](docs/signing/signing_with_containers/)
+* To use Gitsign, see [Sign Git commits with Gitsign](/docs/signing/gitsign/)
+* To learn about verification, see [verify entries with Cosign](/docs/verifying/verify/)
 
 ## Contributing
 
-Up to date documentation, best practices and detailed scenarios for Sigstore live here. These pages are maintained by the community and intended to help anyone get set up easily with any of the technologies, to find what you’re looking for fast. It’s also where we keep all the relevant pages for the Sigstore trust root, from ceremonies to security practices.
+Up to date documentation, best practices and detailed scenarios for sigstore live here. These pages are maintained by the community and intended to help anyone get set up easily with any of the technologies, to find what you’re looking for fast. It’s also where we keep all the relevant pages for the sigstore trust root, from ceremonies to security practices.
```

Comment:
Not your change, but think we should have Oxford comma after "best practices." (We're using Oxford commas, right?)

ceremonies > signing ceremonies

Reply:
+1 for the Oxford comma love